clients/unshaded/src/main/java/tachyon/client/TachyonFile.java

```diff
@@ -495,7 +495,7 @@ boolean recache(int blockIndex) throws IOException {
       inputStream.skip(offset);
 
       int bufferBytes =
-          (int) mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB) * 4;
+          (int) mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB);
```

Comment:
maybe I misread the ticket: "However, the "*4" should probably be with the Constants.MB to be 4 MB"
are you saying the call to getBytes should be something like this?
mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB*4)
making each unit buffer of size 4MB?